### PR TITLE
Spotify api

### PR DESCRIPTION
**Pull request by alexfrey317**
PR for adding client auth to backend.  What I did:

Created a spotify configuration file that uses constants from a file.
TODO: Put constants somewhere safe

### CODE REVIEW

File: src/main/java/com/mixtape/mixtapeapi/MixtapeApiConstants.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.mixtape.mixtapeapi;
+
+public class MixtapeApiConstants {
+    // TODO: Move to secure location
+    public final static String clientId = "df9bd9e5ec41469baf91e29921d605a9";
+    public final static String clientSecret = "1f740b22a8984436bb87e41d7fa23295";
+}
```
Comment (softwaresale) [inline src/main/java/com/mixtape/mixtapeapi/MixtapeApiConstants.java:7]: The secure place to put these is in the config file. I'll show you how to do that

**Review by softwaresale — COMMENTED** [review]
Looks good other than the the properties thing

File: src/main/java/com/mixtape/mixtapeapi/MixtapeApiConstants.java (hunk 1 above)
Comment (softwaresale) [reply src/main/java/com/mixtape/mixtapeapi/MixtapeApiConstants.java:7]: Basically, you're gonna use [configuration properties](https://www.baeldung.com/configuration-properties-in-spring-boot)

Comment (alexfrey317) [reply src/main/java/com/mixtape/mixtapeapi/MixtapeApiConstants.java:7]: I'm merging it and next time we meet you can attach the auth you worked on and set up the config file thing

**Pull request merged**